docs/spec/auth/oauth.md

```diff
@@ -39,10 +39,10 @@ Content-Type: application/x-www-form-urlencoded
         (REQUIRED) Type of grant used to get token. When getting a refresh token
         using credentials this type should be set to "password" and have the
         accompanying username and password paramters. Type "authorization_code"
-        is reserved for future use for authenticating to an authorization server
-        without having to send credentials directly from the client. When
-        requesting an access token with a refresh token this should be set to
-        "refresh_token".
+        is used for authenticating to an authorization using a code given from
+        directly from a resource owner and without having to send credentials
```

Comment:
nit: "given from directly from" -> "provided by"?  And you also want to keep the "server" part of "authorization server".